main.R

```diff
@@ -0,0 +1,26 @@
+library(dplyr)
+library(miceadds)
+library(readr)
+library(stringr)
+library(sbtools)
+library(whisker)
+
+outer_dir <- 'C:/Users/SLEUNG/Github/ds-pipelines-targets-1'
```

Comment:
The hard-coded path will be a problem for others trying to replicate your work. Can you please replace it with a relative path so that anybody who clones this can run the code? That will introduce a requirement that the code be run from a specific directory, but that is OK.